src/benchmarks/micro/coreclr/System.Reflection/Attributes/AttributeTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System;
+using System.Reflection;
+
+namespace Attributes
```

Comment:
could you please call this namespace `System.Reflection`? it's going to allow us for very easy filtering in the future: `--filter System.Reflection*` and also we use `namespace.typeName.methodName` as benchmark ids in our internal reporting tools